encoders/nlp/TFIDFTextEncoder/README.md

```diff
@@ -0,0 +1,113 @@
+# TFIDFTextEncoder
+
+ **TFIDFTextEncoder** is a class that wraps the text embedding functionality from a **TFIDF** model.
+
+The **TFIDF** model is a classic vector representation for  [information retrieval](https://en.wikipedia.org/wiki/Tf–idf).
+
+`TfidfTextEncoder` encodes data from a `np.ndarray` of strings and returns a `scipy.csr_matrix` of floating point values.
+
+- Input shape: `BatchSize `
+
+- Output shape: `BatchSize x EmbeddingDimension`
+
+
+
+
+## How to 
+
+### Dependencies
+
+`TFIDFTextEncoder` is dependend on `sckikit-learn`:
+
+```
+pip install scikit-learn
+```
+
+
+
+### Use as Python Class
+
+Before the `TFIDFTextEncoder` is used a `sklearn.feature_extraction.text.TfidfVectorizer` object needs to be fitted and stored as a pickle object which the `TFIDFTextEncoder` will load. 
+
+The following snipped can be used to fit a `TfidfVectorizer` with a toy corpus from sklearn. To achieve better performance or adapt the encoder to other languages users can change the `load_data` function from below to load any other user specific dataset.
+
+```python
+from sklearn.feature_extraction.text import TfidfVectorizer
+import pickle
+
+def load_data():
+    from sklearn.datasets import fetch_20newsgroups
+    newsgroups_train = fetch_20newsgroups(subset='train')
+    return newsgroups_train.data
+
+if __name__ == '__main__':
+    X = load_data()    
+    tfidf_vectorizer = TfidfVectorizer()
+    tfidf_vectorizer.fit(X)
+    pickle.dump(tfidf_vectorizer, open("tfidf_vectorizer.pickle", "wb"))
```

Comment:
I need information about how to use the `TFIDFTextEncoder` and not the `TFIDFVectorizer`.

You can show how from this pickle u can load a `TFIDFTextEncoder`

Reply:
please use single quote